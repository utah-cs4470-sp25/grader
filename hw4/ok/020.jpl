struct pair {
  x: int
  y: float
}

let mypair = pair{1, 2}
let myx = mypair.x
let myy = mypair.y
