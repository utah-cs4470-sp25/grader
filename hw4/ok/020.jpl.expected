(StructCmd pair x (IntType) y (FloatType))
(LetCmd (VarLValue mypair) (StructLiteralExpr pair (IntExpr 1) (IntExpr 2)))
(LetCmd (VarLValue myx) (DotExpr (VarExpr mypair) x))
(LetCmd (VarLValue myy) (DotExpr (VarExpr mypair) y))
